src/main/resources/lang/default.lang

```diff
@@ -1313,6 +1313,18 @@ teleport causes:
 	unknown: unknown
 	dismount: dismount, dismounted
 
+# -- Inventory Close Reasons --
+inventory close reasons:
+	unknown: unknown
+	teleport: teleport
+	cant_use: cant use, can not use
```

Comment:
can we not use `can't` here? "cant" isn't the right word. Might also be worth adding "cannot".